thirdparty/openapi3/schemas/UpdateCredentialRequest.yaml

```diff
@@ -0,0 +1,28 @@
+title: UpdateCredentialRequest
```

Comment:
Really confused about this definition. It doesn't be seem to be a request but more akin to a property used in `ConsentIDPutRequest`  https://github.com/mojaloop/api-snippets/pull/17/files#diff-2b332d878ba42f73f319a12fb3fccfb4R22 .

Maybe something like CredentialProof? @Auth-Team

Reply:
I think this was my fault - and there's a good chance I did something wrong here. 

One of the challenges/quirks with the api design is that we have a ton of fields which could be null or undefined. My goal with request bodies (or parts of request bodies) was to have a schema which would be very explicit, and thus make writing the types and code much easier.

Renaming it from `...Request` would make this less confusing - `CredentialProof` seems suitable, or perhaps `UpdatedCredential`

Reply:
UpdatedCredential sounds good. done.